kitchenci/.kitchen.yml

```diff
@@ -0,0 +1,16 @@
+---
+driver:
+  name: docker
+
+provisioner:
+  name: puppet_apply
+  puppet_version: "3.8.1"
```

Comment:
Ok, I get that. but should it be better to a parameter similar to the one you have used for platforms place holder?